### ISSUE TITLE
Make it clear that a story is in the creation phase

### ISSUE THREAD

**Issue by hannesroth88** [Dream-beez/issue-tracker#34]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When you create a new Story and leave the loading screen. In your list of stories the story to be created is shown as "Title" and there's no content.

### Expected Behavior

Would advice to make it clear that it's still beeing created.
Something like "in progress" in the title. 

### Smartphone

```markdown
- Device:
- OS:
- App Version:
```

**Comment by frank-bee** [comment]
Good point, we anyway want to change this video and had this issue in mind .
Thanks
